schemas/vsds.json

```diff
@@ -362,6 +362,36 @@
                 "propertyOrder": 410,
                 "sectionEnd": "Security and Certificates",
                 "advanced": true
+            },
+            "vsd_ram": {
+                "type": "integer",
+                "title": "KVM VSD RAM",
```

Comment:
Can we change the title to just VSD RAM instead of KVM VSD RAM? I know it is KVM VSD Ram in the common,  but that title was a mistake. We cannot change that without breaking backward compatibility, but we are adding something new here, so we can change this. Also please regenerate the examples after doing that. Change the titles got CPU cores as well